### DISCUSSION TITLE
URL parameters without value (../?asdf)

### DISCUSSION THREAD

**Discussion by seimnseimn** [projectdiscovery/httpx#899] [Q&A]
Dear gents,

I could not find anything in SO or this repo.
According to [SO: Is a url query parameter valid if it has no value?](https://stackoverflow.com/a/4557763/10816481) it's ok to have a query param with no value. E.g. `.../?bla` (I don't mean `.../?bla=` or `.../?bla=None`, etc.)

However I could not find a way to do this with httpx `params=`

It this anyhow possible?

**Comment by olearycrew** [comment]
Could you do this with a raw request?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
